wagesAnalysis.sh

```diff
@@ -0,0 +1,31 @@
+### Script for getting information about highest and lowest earning people
+### Usage: bash wagesAnalysis.sh
+
+#
+echo "Highest earner:"
+### Gets the gender, yearsExperience, and wages of the highest earner
+cat wages.csv | sed 's/,/ /g' | sort -k 4 -n | tail -1 | cut -d " " -f1,2,4 
+
+#
+echo "Lowest earner:"
+### Gets the gender, yearsExperience, and wages of the lowest earner
+cat wages.csv | sed 's/,/ /g' | sort -k 4 -n | head -2 | grep -v "gender" | cut -d " " -f1,2,4 
+
+#
+echo "Number of females in the top 10 earners:"
+cat wages.csv | sed 's/,/ /g' | sort -k 4 -n | grep -v "gender" | head -10 | cut -d " " -f1,2,4 | grep "female"| wc -l 
+
+#
+echo "Average wage of those with 12 years school:"
+cat wages.csv | cut -d "," -f 3,4 | grep "12," | sed 's/,/ /g' | awk '{ sum += $2; n++ } END { if (n > 0) print sum / n; }' 
+
+#
+echo "Average wage of those with 16 years school:"
+cat wages.csv | cut -d "," -f 3,4 | grep "16," | sed 's/,/ /g' | awk '{ sum += $2; n++ } END { if (n > 0) print sum / n; }' 
+
+#
+echo "Difference in wage:"
+VAR1="$(cat wages.csv | cut -d "," -f 3,4 | grep "16," | sed 's/,/ /g' | awk '{ sum += $2; n++ } END { if (n > 0) print sum / n; }')"
+VAR2="$(cat wages.csv | cut -d "," -f 3,4 | grep "12," | sed 's/,/ /g' | awk '{ sum += $2; n++ } END { if (n > 0) print sum / n; }')"
+
+echo "$VAR1 - $VAR2" | bc
```

Comment:
I think the question was asking for the difference in minimum wage between genders, but average is OK. Kudos for incorporating awk!